### PR TITLE
upgrades: Quote dsn printing in log message

### PR DESCRIPTION
**Pull request by efritz**
Koow when an empty dsn

## Test plan

yes you do

### CODE REVIEW

**Review by DaedalusG — APPROVED** [review]
lgtm

**Comment by DaedalusG** [comment]
backport?

File: internal/database/connections/live/runner.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func RunnerFromDSNsWithSchemas(out *output.Output, logger log.Logger, dsns map[s
 				pending.Destroy()
 				return nil, err
 			}
-			pending.Complete(output.Emojif(output.EmojiSuccess, "Connection to %s succeeded", dsns[name]))
+			pending.Complete(output.Emojif(output.EmojiSuccess, "Connection to %q succeeded", dsns[name]))
```
Comment (Strum355) [inline internal/database/connections/live/runner.go:42]: this doesnt get filled in with the default value? :thonk:

Comment (efritz) [reply internal/database/connections/live/runner.go:42]: Trying to see if either `dsns[name] == '` or if the key is missing are possibilities.

**Pull request merged**